lib/mcb.rb

```diff
@@ -19,6 +19,8 @@ def self.init_rails
     unless defined?(Rails)
       app_root = File.expand_path(File.join(File.dirname($0), '..'))
       exec_path = File.join(app_root, 'bin', 'rails')
+      ENV["DISABLE_SPRING"] = "true" # prevent caching of environment variables by spring
```

Comment:
Not sure what the rest of the team think but I’d be happy to remove spring completely

Reply:
Spring can be a PITA, but it does reduce start-up time. I'm on the fence on this one.

Reply:
Does rspec use Spring? If so, I'd be really tempted to keep it.